MapboxDirections/MBDirectionsOptions.swift

```diff
@@ -404,6 +409,13 @@ open class DirectionsOptions: NSObject, NSSecureCoding, NSCopying {
      */
     @objc open var includesVisualInstructions = false
     
+    /**
+     Indicate how a route considers from which side of the road to approach a waypoint.
```

Comment:
This is a property, not an action method, so describe the effect rather than an action.

> A Boolean value that indicates whether each of the resulting routes must approach a waypoint from the same side of the road.